antenna/breakpad_resource.py

```diff
@@ -150,10 +151,10 @@ def check_health(self, state):
 
     def hb_health_stats(self):
         # The number of crash reports sitting in the queue
-        self.mymetrics.gauge('save_queue_size', len(self.save_queue))
+        self.mymetrics.gauge('save_queue_size', len(self.crashmover_save_queue))
 
         # The number of actively running coroutines saving crashes
-        self.mymetrics.gauge('active_save_workers', len(self.pipeline_pool))
+        self.mymetrics.gauge('active_save_workers', len(self.crashmover_pool))
```

Comment:
We don't need this anymore. It's either 0 or 1 and most of the time when measured it'll probably be 0. So ... that doesn't tell us anything useful.